apps/explorer/src/components/validator-map/index.tsx

```diff
@@ -85,23 +76,20 @@ export default function ValidatorMap({ minHeight }: Props) {
         return count;
     }, [nodeData]);
 
-    useEffect(() => {
-        setIsLoading(true);
-        appsBe(`validator-map`, {
-            network: network.toLowerCase(),
-        })
-            .then((res) => {
-                const data = res as ValidatorMapData[];
-                setValidatorCount(data.length);
-                // Some validators will come back as null from the API
-                const validatorResponse = data.filter((validator) => validator);
-                setValidatorData(validatorResponse);
-                setIsLoading(false);
-            })
-            .catch((err) => {
-                setIsError(true);
-            });
-    }, [appsBe, network]);
+    const {
+        data: validatorData,
+        isLoading,
+        isError,
+    } = useQuery({
+        queryKey: 'validator-map' as unknown as QueryKey,
```

Comment:
getting linting errors for some reason here, so using the object syntax

Reply:
This should be in an array?